mkosi/kmod.py

```diff
@@ -11,7 +11,9 @@
 from mkosi.run import bwrap, chroot_cmd
 
 
-def filter_kernel_modules(root: Path, kver: str, include: Sequence[str], exclude: Sequence[str]) -> list[Path]:
+def filter_kernel_modules(
+    root: Path, kver: str, include: Sequence[str], exclude: Sequence[str]
```

Comment:
Single line per argument here and elsewhere?

Reply:
I'd prefer to keep it this way, since black only moves to single line per argument if the line length goes over the column limit. In the hope that we can one day go back to an autoformatter, I'd like to not introduce additional churn here.

Reply:
This is inconsistent with how we do it already though, I'd prefer us to use single argument per line everywhere as soon as we have to start wrapping things

Reply:
Not necessary anymore with the limit at 119.